src/infrastructure/implementations/blockstack-crux-user-repository.ts

```diff
@@ -66,10 +68,11 @@ export class BlockstackCruxUserRepository implements ICruxUserRepository {
             return;
         }
         let addressMap = {};
-        let cruxUserData: ICruxUserData = {
+        let cruxUserData: any = {
```

Comment:
Do we need to switch return type here from interface to any?

Reply:
https://github.com/cruxprotocol/js-sdk/pull/164/files#diff-d2eec6f48915539797e3d7d7d4020062R76